<ui version="4.0" >
 <author>GeometryFactory</author>
 <class>Delaunay_triangulation_2</class>
 <widget class="QMainWindow" name="Delaunay_triangulation_2" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CGAL Delaunay Triangulation</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/CGAL.qrc" >
    <normaloff>:/cgal/logos/cgal_icon</normaloff>:/cgal/logos/cgal_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QGraphicsView" name="graphicsView" >
      <property name="focusPolicy" >
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy" >
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy" >
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="transformationAnchor" >
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="fileToolBar" >
   <property name="windowTitle" >
    <string>File Tools</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear" />
   <addaction name="actionLoadPoints" />
   <addaction name="actionSavePoints" />
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>Visualization Tools</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionSetColorGreen" />
   <addaction name="actionSetColorRed" />
   <addaction name="actionSetColorBlue" />
   <addaction name="actionSetColorBlack" />
   <addaction name="separator" />
   <addaction name="actionInsertPoint" />
   <addaction name="actionSelectPoint" />
   <addaction name="separator" />
   <addaction name="actionRecenter" />
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="separator" />
    <addaction name="actionClear" />
    <addaction name="actionLoadPoints" />
    <addaction name="actionSavePoints" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionInsertRandomPoints" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSetColorGreen" />
    <addaction name="actionSetColorRed" />
    <addaction name="actionSetColorBlue" />
    <addaction name="actionSetColorBlack" />
    <addaction name="separator" />
    <addaction name="actionInsertPoint" />
    <addaction name="actionSelectPoint" />
    <addaction name="separator" />
    <addaction name="actionRecenter" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuTools" />
  </widget>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutCGAL" >
   <property name="text" >
    <string>About &amp;CGAL</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInsertRandomPoints" >
   <property name="text" >
    <string>&amp;Insert random points</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSetColorRed" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="Delaunay_triangulation_2.qrc" >
     <normaloff>:/cgal/Actions/icons/inputPointRed.png</normaloff>:/cgal/Actions/icons/inputPointRed.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;set color to red</string>
   </property>
   <property name="toolTip" >
    <string comment="The comment" >set color to red</string>
   </property>
   <property name="statusTip" >
    <string comment="and its comment" >Nothing to say</string>
   </property>
   <property name="whatsThis" >
    <string comment="what" >whats this</string>
   </property>
  </action>
  <action name="actionSetColorGreen" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="Delaunay_triangulation_2.qrc" >
     <normaloff>:/cgal/Actions/icons/inputPointGreen.png</normaloff>:/cgal/Actions/icons/inputPointGreen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;set color to green</string>
   </property>
   <property name="toolTip" >
    <string>set color to green</string>
   </property>
   <property name="statusTip" >
    <string>set color to green</string>
   </property>
  </action>
  <action name="actionClear" >
   <property name="icon" >
    <iconset resource="../icons/File.qrc" >
     <normaloff>:/cgal/fileToolbar/fileNew.png</normaloff>:/cgal/fileToolbar/fileNew.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSelectPoint" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons/Triangulation_2.qrc" >
     <normaloff>:/cgal/Triangulation_2/Voronoi_diagram_2.png</normaloff>:/cgal/Triangulation_2/Voronoi_diagram_2.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Select point</string>
   </property>
  </action>
  <action name="actionInsertPoint" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="Delaunay_triangulation_2.qrc" >
     <normaloff>:/cgal/Actions/icons/triangulation.png</normaloff>:/cgal/Actions/icons/triangulation.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Insert point</string>
   </property>
  </action>
  <action name="actionLoadPoints" >
   <property name="icon" >
    <iconset resource="../icons/File.qrc" >
     <normaloff>:/cgal/fileToolbar/fileOpen.png</normaloff>:/cgal/fileToolbar/fileOpen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Load Points...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSavePoints" >
   <property name="icon" >
    <iconset resource="../icons/File.qrc" >
     <normaloff>:/cgal/fileToolbar/fileSave.png</normaloff>:/cgal/fileToolbar/fileSave.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save Points...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSetColorBlue" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="Delaunay_triangulation_2.qrc" >
     <normaloff>:/cgal/Actions/icons/inputPointBlue.png</normaloff>:/cgal/Actions/icons/inputPointBlue.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;set color to blue</string>
   </property>
   <property name="toolTip" >
    <string>Set color to blue</string>
   </property>
  </action>
  <action name="actionSetColorBlack" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="Delaunay_triangulation_2.qrc" >
     <normaloff>:/cgal/Actions/icons/inputPointBlack.png</normaloff>:/cgal/Actions/icons/inputPointBlack.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;set color to black</string>
   </property>
   <property name="toolTip" >
    <string>set color to black</string>
   </property>
  </action>
  <action name="actionRecenter" >
   <property name="icon" >
    <iconset resource="../icons/Input.qrc" >
     <normaloff>:/cgal/Input/zoom-best-fit</normaloff>:/cgal/Input/zoom-best-fit</iconset>
   </property>
   <property name="text" >
    <string>Re&amp;center the viewport</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Delaunay_triangulation_2.qrc" />
  <include location="../icons/File.qrc" />
  <include location="../resources/CGAL.qrc" />
  <include location="../icons/Triangulation_2.qrc" />
  <include location="../icons/Input.qrc" />
 </resources>
 <connections/>
</ui>
